anchor/Anchor.toml

```diff
@@ -19,7 +19,7 @@ cluster = "localnet"
 wallet = "~/.config/solana/id.json"
 
 [scripts]
-test = "../node_modules/.bin/nx run anchor:jest"
+test = "../node_modules/.bin/pnpm run test --filter=anchor"
```

Comment:
Suggest changing to 
```
test = "../node_modules/.bin/jest ./tests/jest --config ./jest.config.ts"
```
as at least for me there is no binary at `./node_modules/.bin/pnpm`.

Reply:
To verify that tests are running run:

`git submodules update --init` (if haven't done previously)

and while wd is `./anchor` run

`pnpm run test`

Reply:
anchor folder there is a jest.config.ts, in which the deleted jest.preset.js is still referenced, please remove that config line.